src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -98,21 +98,23 @@ public static void AddFilesToProject(this Project project, IEnumerable<string> f
             var solutionService = Package.GetGlobalService(typeof(SVsSolution)) as IVsSolution;
 
             IVsHierarchy hierarchy = null;
-            solutionService?.GetProjectOfUniqueName(project.UniqueName, out hierarchy);
-
-            if (hierarchy == null)
-                return;
-
-            var ip = (IVsProject)hierarchy;
-            VSADDRESULT[] result = new VSADDRESULT[files.Count()];
+            if (solutionService != null && ErrorHandler.Succeeded(solutionService.GetProjectOfUniqueName(project.UniqueName, out hierarchy)))
+            {
+                if (hierarchy == null)
+                    return;
+
+                var ip = (IVsProject)hierarchy;
```

Comment:
the name `ip` seems misleading.  I know it was pre-existing, but renaming it would make this code more understandable IMO.